pkg/oomstore/online_query.go

```diff
@@ -108,11 +114,17 @@ func (s *OomStore) getFeatureValueMap(ctx context.Context, entityKeys []string,
 		if len(features) == 0 {
 			continue
 		}
+
+		group := features[0].Group
+		if group.Category == types.CategoryBatch && group.OnlineRevisionID == nil {
+			return nil, errdefs.Errorf("the OnlineRevision of the group %s is nil,please check if there is no sync yet", group.Name)
```

Comment:
group %s has nil OnlineRevision, please make sure you have already `sync`-ed

Reply:
done